stylelint-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-standard",
+  "plugins": ["stylelint-scss", "stylelint-order"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true
+  }
```

Comment:
There are some pure CSS files around:

```suggestion
  "overrides": [
    {
      "files": ["*.scss", "**/*.scss"],
      "rules": {
        "at-rule-no-unknown": null,
        "scss/at-rule-no-unknown": true
      }
    }
  }
```

Reply:
Is this just so we have a place to put scss specific rules? Because without the override it still successfully lints all CSS/SCSS files if you can `yarn stylelint "**/*.{css,scss,sass}"`

Reply:
Gotcha 👍